source/write-to-mongodb.txt

```diff
@@ -28,6 +28,18 @@ Write to MongoDB
 
          .. include:: /scala/write-to-mongodb.txt
 
+.. warning::
+
+   If you specify the ``overwrite`` save mode, the connector drops the target
```

Comment:
Q: is this a write mode? I saw this option referred to as a write mode in this [ticket](https://jira.mongodb.org/browse/SPARK-364)

```suggestion
   If you specify the ``overwrite`` write mode, the connector drops the target
```